README.md

```diff
@@ -62,10 +61,32 @@ sam package --template-file .aws-sam/build/template.yaml \
 aws s3 cp .aws-sam/build/lambda-template.yaml s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/lambda-template/master/
 ```
 
+## Publish Lambda
+
+### Private access
+Publishing the lambda makes it available in your AWS account.  It is accessible in
+the [serverless application repository](https://console.aws.amazon.com/serverlessrepo).
+
+```shell script
+sam publish --template .aws-sam/build/lambda-template.yaml
```

Comment:
I believe a user of this template would have to replace `lambda-template.yaml` with the name of their own repository, correct? I don't think that's clear from this README

Reply:
not really, it's the template file specified with the `--output-template-file` flag in the  previous `sam package` command.  We have designed our deploy strategy to have matching repo name and output templates.

We can expect lambda devs to have a basic understanding of the SAM framework and realize that.